css/landing.css

```diff
@@ -30,13 +31,13 @@
   color: black;
   font-size: 24px;
   line-height: 30px;
-  margin: 0 30px;
+  margin: 35px 30px;
   font-family: Rowdies;
 }
 
 .start-btn {
```

Comment:
Add `cursor: pointer;` to the start button and add some `margin-bottom` to it

![image](https://user-images.githubusercontent.com/78752405/144186304-25f53c39-2b08-41f5-888a-e31acbd3cbfb.png)